<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bingrenxinxi</class>
 <widget class="QDialog" name="bingrenxinxi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>病人信息</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <property name="locale">
   <locale language="Chinese" country="China"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>470</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>床号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>60</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>身高 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>体重</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>血型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>病历号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>医生姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>病人姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>出生日期 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>入院日期</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_xingbie">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>20</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_leixing">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_xuexing">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>100</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>5</number>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <property name="duplicatesEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_binglihao">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_bingrenxingming">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yishengxingming">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>300</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_shengao">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>150</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_tizhong">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>500</number>
   </property>
   <property name="value">
    <number>60</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>66</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>cm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>kg</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_chuanghao">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_chusheng_nian">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1900</number>
   </property>
   <property name="maximum">
    <number>2200</number>
   </property>
   <property name="value">
    <number>1970</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ruyuan_nian">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2015</number>
   </property>
   <property name="maximum">
    <number>2200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>275</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_chusheng_yue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>275</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_chusheng_ri">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>31</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>275</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>315</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>315</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>315</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ruyuan_yue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ruyuan_ri">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>310</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>31</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
